tbx/people/models.py

```diff
@@ -56,21 +63,22 @@ class PersonPage(ColourThemeMixin, SocialFields, Page):
         + ColourThemeMixin.promote_panels
         + [
             MultiFieldPanel(SocialFields.promote_panels, "Social fields"),
+            FieldPanel("related_teams", widget=forms.CheckboxSelectMultiple),
         ]
     )
 
     @cached_property
     def author_posts(self):
-        # return the blogs writen by this member
         author_snippet = Author.objects.get(person_page__pk=self.pk)
 
-        # format for template
+        # Format for template
         return [
             {
                 "title": blog_post.title,
                 "url": blog_post.url,
                 "author": blog_post.first_author,
                 "date": blog_post.date,
+                "tags": blog_post.tags,
```

Comment:
@lparsons396 can you add the reading time here?

Reply:
I've also updated this to exclude the author but include the read_time.